src/System.Net.Security/tests/Scripts/setup-kdc.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+
+OS=`cat /etc/os-release | grep "^ID=" | sed 's/ID=//g' | sed 's/["]//g' | awk '{print $1}'`
+echo "Operating System: ${OS}"
+
+realm="TEST.COREFX.NET"
+
+principal1="HOST/host.test.corefx.net"
+principal2="HTTP"
+krb_user="krb_user"
+password="password"
+
+kadmin="kadmin.local"
+krb5kdc="krb5kdc"
+kdb5_util="kdb5_util"
+add_principal_cmd="add_principal -pw ${password}"
+
+krb_conf="krb5.conf"
+krb_conf_location="/etc/"
+keytabfile="/etc/krb5.keytab"
+
+# Common function across linux distro's to configure KDC post installation
+configure_kdc()
+{
+	echo "Stopping KDC.."
+	if pgrep krb5kdc 2> /dev/null; then sudo killall krb5kdc ; fi
+	
+	# Remove database files if exist
+	# There are multiple kdc db files with name pattern principal*. Doing "sudo rm -f" doesn't work,
+	# because the file name expansion happens in context of normal user, (who does not have pemission on the path)
+	# and then the command is passed to sudo. Hence it tries to delete file with name "principal*", which does not exist.
+	# To overcome this and make file name expansion under sudoer/superuser, rm is run under a separate bash 
+	sudo bash -c "rm -f ${database_files}"
+
+	# Create/copy krb5.conf and kdc.conf
+	echo "Copying krb5.conf and kdc.conf.."
+	sudo /bin/cp ${krb_conf} ${krb_conf_location}
+	sudo /bin/cp ${kdc_conf} ${kdc_conf_location}
+	
+	echo "Creating KDC database for realm ${realm}.."
+	sudo ${kdb5_util} create -r ${realm} -P ${password} -s
```

Comment:
What if one of these steps fails?  Should we be checking for process exit codes and erroring out if it's not successful?